src/components/HrDivider/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
Ok a few things here:
1. The Component should be named `HrDivider` although that name doesn't really tell me about what the component does. 
2. Do you think we'll re-use this component? If not, we should just put this directly on the page.